doc/doc-primitives.saty

```diff
@@ -59,9 +63,43 @@ document (|
       どのフォントで組まれるかや「空白がどの程度伸縮できるか」「どこで改ページしてよいか」といった
       情報も保持している．
     }
+    +type (tMATH) {
+      数式を扱う型．
+    }
     +type (tCTX) {
       コンテキスト型．組版処理に必要な情報のうちテキスト以外のものをすべて保持している．
     }
+    +type (tTCTX) {
+      テキスト出力モードで使用されるコンテキスト型．
+    }
+    +type (tPT) {
+      ${\paren{x, y}}で座標を表す．
+      左下が原点で、右に行くほど${x}の値が大きくなり，上に行くほど${y}の値が大きくなる座標系である．
```

Comment:
The characters `，` and `、` seems coexisting. It is preferable to use `，` only in this document. The character `、` also occurs at other 4 places.